ci-operator/step-registry/openshift/e2e/ibmcloud/ovn/openshift-e2e-ibmcloud-ovn-workflow.yaml

```diff
@@ -10,6 +10,7 @@ workflow:
     post:
     - chain: gather-network
     - chain: gather-core-dump
+    - ref: gather-ibmcloud-resource
```

Comment:
how about add gather-ibmcloud-resource before destroy cluster , I add it in https://github.com/openshift/release/blob/fa816aa4e57174d36c2617cdfbdba6c8ed265e8c/ci-operator/step-registry/cucushift/installer/rehearse/ibmcloud/ipi/deprovision/cucushift-installer-rehearse-ibmcloud-ipi-deprovision-chain.yaml#L6

Reply:
I may be wrong, as the step-registry gets a bit cumbersome to track through, but I think the current workflow does not call into `ibmcloud-ipi-deprovision` chain, just `ipi-deprovision` (below) chain, which I believe is platform agnostic

Unless I should make changes to this and other workflows to use `ibmcloud-ipi-deprovision` as well....assuming I followed the breadcrumbs properly.